backend/src/test/java/com/soen341/instagram/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,71 @@
+package com.soen341.instagram.service;
+
+
+import com.soen341.instagram.dao.impl.AccountRepository;
+import com.soen341.instagram.exception.account.EmailTakenException;
+import com.soen341.instagram.exception.account.InvalidEmailFormatException;
+import com.soen341.instagram.exception.account.InvalidUsernameFormatException;
+import com.soen341.instagram.exception.account.UsernameTakenException;
+import com.soen341.instagram.model.Account;
+import com.soen341.instagram.service.impl.RegistrationService;
+import org.junit.jupiter.api.Test;
```

Comment:
Just a picky comment: would it be possible to add an empty line between the  `com.` and `org.` imports? Since they originate from two different base packages (?)